app/posts/apply-for-teacher-training/2021-07-23-end-of-cycle-2021.md

```diff
@@ -0,0 +1,226 @@
+}) }}
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "apply-again-next-cycle.png"
+  },
+  caption: "If the application does not lead to a place when the deadline to apply again has already passed, the usual 'apply again' email changes to reflect this."
+}) }}  
+
+[If the candidate carries their application over before Apply reopens, they cannot submit](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-apply-reopens%2C-they-cannot-submit) - as above.
+
+[If the candidate carries their application over before Find reopens, they cannot find courses](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-find-reopens%2C-they-cannot-find-courses) - as above.
+
+## Candidates who create an account after the deadline to apply for the first time
+
+[If the candidate creates an account before Apply reopens, they cannot submit](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-apply-reopens%2C-they-cannot-submit) - as above.
+
+[If the candidate creates an account before Find reopens, they cannot find courses](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-find-reopens%2C-they-cannot-find-courses) - as above.
+
+## Timeframe to carry over an application
+
+We have not set an end point for carrying over an application.
+
+This means that the [carry over message when a candidate's application does not lead to a place](/apply-for-teacher-training/end-of-cycle-2021/#if-it%E2%80%99s-too-late-to-apply-in-the-same-recruitment-cycle%2C-they-need-to-%E2%80%98carry-over%E2%80%99-their-application) and the [unsubmitted carry over message](/apply-for-teacher-training/end-of-cycle-2021/#they-need-to-%E2%80%98carry-over%E2%80%99-their-application-to-continue-working-on-it) continue to show indefinitely until the candidate actions it.
+
+Wherever we refer to applying for courses starting in a particular year, such as in carry over messages or banners, it should always reflect the year the candidate is eligible to start a course in.
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "carry-over-year-later.png"
+  },
+  caption: "Example: if a candidate started an application when it was possible to get on a course starting in the 2021 to 2022 academic year, but does not submit in time and does not sign back in until July 2023."
+}) }}  
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "application-unsuccessful-no-longer-possible-apply-again-year-later.png"
+  },
+  caption: "Example: if a candidate submitted an application when it was possible to get on a course starting in the 2021 to 2022 academic year, but does get a place in time and does not sign back in until July 2023."
+}) }}  
+
+We're assessing whether we should set an end point.
```

Comment:
Ahhh.

Reply:
ohhhh.... hahah

Reply:
i'll bring that sentence further up so it's clearer haha